src/crate/client/http.py

```diff
@@ -88,6 +90,8 @@ class CrateJsonEncoder(json.JSONEncoder):
     def default(self, o):
         if isinstance(o, Decimal):
             return str(o)
+        if isinstance(o, UUID):
+            return str(o)
```

Comment:
Might be a slight performance improvement, summing up when sending many requests.

```suggestion
        if isinstance(o, (Decimal, UUID)):
            return str(o)
```